src/main/scala/com/bazaarvoice/sswf/SSWFStep.scala

```diff
@@ -0,0 +1,19 @@
+package com.bazaarvoice.sswf
+
+/**
+ * Your Step Enum must implement/extend this
+ */
+trait SSWFStep {
+  /** The name of the Step */
+  def name: String
+  /** The version of the step definition (increment if you change any options) */
```

Comment:
I figured out how to remove this version, but I also updated the readme to discuss versioning strategy as well as other concepts like task lists.